README.md

```diff
@@ -7,51 +7,53 @@ This project was built using only built in web services (ie no NPM packages besi
+
+The Windows OS provides some fonts like "Wingdings 2" and some specific styles like "Sitka Heading" that do not necessarily map to fonts available, by default, on the Web. This is important to consider when using the Local Font Access API to be sure to set up fallbacks or handle cases where the font does not map.
+
+### User permission
+The above two API's both will require user permission. I consider this the double edge sword of the web. For developers, you have to consider the case where the user does not allow access for the necessary API. Sometimes the API is central to the app functioning, like the File System API in this PWA so if this were to be a _real_ app, we would probably make sure this was accepted before moving forward. Requesting permission to access certain information it gives the user full transparency and control over what the app is reaching into on their computer which can transfer to the user being more comfortable using a PWA.
+
+This is a difference between PWA and Native applications that can not be avoided. This should no be viewed as a strictly negative thing because it does indeed benefit the user.
 
 ### 🎯 Selection API
-- Find (and the unimplemented Replace)
-- Column, Col positions
-- Copy, Cut, Paste, Select all
-- Limitations
-  - Gets really tricky with `shadowRoot`. Supported in chromium but not everywhere
-
-#### 🗄️ Local Storage
-- Saving state
-  - Notepad text
-  - Settings
-  - Last search string in the find/replace bar
-- Limitations
-  - Making sure this isn't bogging down someones browser when things get LARGE (is this real?)
+
+Notepad has a set of "Edit" functions that do an array of different text insertion, deletion and searching. In order to figure out what portion of the editor the user is targeting, the Selection API gives the "range" of highlighted text or the singular cursor position. This allows the developer know where to "Time/Date" or where to paste text. For find/replace, we searched beginning and end of the desired substrings within the editor and then used the Selection API to create a range (which highlights the text) at the given start and end point. Once highlighted, we can control whatever text belongs in the range to replace, cut, copy or delete.
+
+The framework being used caused for some hiccups when getting started with this API. Lit is a web components frame work where you can create these smaller building block components and bring them together to create a full webpage and when nested create `shadowRoots` that have thier own scope (per component). This causes functions that are normally called like `window.getSelection()` to not return what the documentation says you should expect because things are buried within shadowRoots. In chromium, `shadowRoot.getSelection()` is supported and due to this, the Notepad PWA will only work in chromium environments since find/replace and all the edit functions are core to the experience.
```

Comment:
```suggestion
The framework being used caused some hiccups when getting started with this API. Lit is a web component framework where you can create these smaller building block components and bring them together to create a full webpage and when nested create `shadowRoots` that have their own scope (per component). This causes functions that are normally called like `window.getSelection()` to not return what the documentation says you should expect because things are buried within shadowRoots. In chromium, `shadowRoot.getSelection()` is supported and due to this, the Notepad PWA will only work in chromium environments since find/replace and all the edit functions are core to the experience.
```